SampleShared/Samples/AppRemoting/Scripts/AppRemotingSample.cs

```diff
@@ -1,14 +1,14 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+using Microsoft.MixedReality.OpenXR.Remoting;
 using System.Collections.Generic;
 using System.Net;
 using System.Net.NetworkInformation;
 using System.Net.Sockets;
 using UnityEngine;
-using UnityEngine.XR;
 using UnityEngine.UI;
-using Microsoft.MixedReality.OpenXR.Remoting;
+using UnityEngine.XR;
```

Comment:
Is there any automated way you organize these imports? I was looking in vscode and couldn't find a setting for this that worked

Reply:
Ahh, mainly in the right click menu in full Visual Studio 
![image](https://user-images.githubusercontent.com/3580640/208777826-916e6bd8-a731-40fe-ac58-08b8dda81391.png)

I'm not sure of a way in vscode :/